plyer/platforms/android/qrbarcodereader.py

```diff
@@ -0,0 +1,21 @@
+from jnius import cast
+from jnius import autoclass
+from plyer.facades import QrBarcodeReader
+
+PythonActivity = autoclass('org.renpy.android.PythonActivity')
+Intent = autoclass('android.content.Intent')
+Uri = autoclass('android.net.Uri')
+
+class AndroidQrBarcodeReader(QrBarcodeReader):
+
+    def _scan(self):
+
+        intent = Intent()
+        intent.setAction(Intent.ACTION_VIEW)
+        intent.setData(Uri.parse('http://www.onlinebarcodereader.com'))
```

Comment:
I'm not sure if [Zxing](https://github.com/zxing/zxing) is the right choice because it's java. And the python thing you could find is only a wrapper (you'll need the java code to build). And that won't be portable on other platforms like... at all this way.